plugins/rapture/nexus-rapture-plugin/src/main/resources/static/rapture/NX/view/drilldown/Drilldown.js

```diff
@@ -359,9 +354,7 @@ Ext.define('NX.view.drilldown.Drilldown', {
     }
 
     // Show the proper card
-    items[index].getLayout().setActiveItem(1);
-
-    me.activeModes[index] = items[index].getLayout().getActiveItem();
+    items[index].setCurrentMode(1);
```

Comment:
this seems to indicate that there are never more than 2 levels in a drilldown? In which case it might make the code clearer to use more descriptive vars, i.e. LIST = 0, DETAIL = 1